roles/telemetry_verify_metrics/defaults/main.yml

```diff
@@ -2,5 +2,7 @@
 telemetry_verify_metrics_metric_sources_to_test:
   - ceilometer_compute_agent
   - ceilometer_central_agent
+  - ceilometer_ipmi_agent
   - node_exporter
   - rabbitmq
+  - kepler
```

Comment:
Since the power monitoring tests should run only on NFV machines, could you remove the ceilometer_ipmi_agent and kepler from the default?

Reply:
Removed kepler and ipmi agent from default deployments.